web/components/Dashboard/WantezList/WantezListItem.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { getExperienceLevel } from '../../../lib/experienceLevel';
```

Comment:
you can use absolute paths
```suggestion
import { getExperienceLevel } from 'lib/experienceLevel';
```